Riot/Modules/directory/role/ActRoleViewController.swift

```diff
@@ -0,0 +1,145 @@
+//
+//  ActRoleViewController.swift
+//  Riot
+//
+//  Created by Naurin Afrin on 31/7/20.
+//  Copyright © 2020 matrix.org. All rights reserved.
+//
+
+import UIKit
+
+
+private enum SectionType: Int, CaseIterable {
+    case FILTER = 0
+    case ROLE_CELL = 1
+    
+    var cellIdentifier: String {
+        switch self {
+        case .FILTER:
+            return "RoleFilterTableViewCell"
+        case .ROLE_CELL:
+            return "RoleTableViewCell"
+        }
+    }
+}
+
+class ActRoleViewController: UIViewController {
+    @IBOutlet weak var recentsTableView: UITableView!
+    
+    var roles = [
+        RoleModel(name: "Test1", description: "Emergency Department", isExpanded: false),
```

Comment:
Please add a TODO comment that this should be replaced once the back end is available

Reply:
Added TODO message at this point.